### PR TITLE
[Static Linux] Add build scripts.

### PR DESCRIPTION
**Pull request by al45tair**
Add the build scripts for the fully static Linux SDK.

rdar://129171568

### CODE REVIEW

**Comment by al45tair** [comment]
@swift-ci test

File: swift-ci/sdks/static-linux/scripts/fetch-source.sh [hunk 1]
```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+#
+# ===----------------------------------------------------------------------===
+#
+#  Swift Static SDK for Linux: Fetch Sources
+#
+#  This source file is part of the Swift.org open source project
+#
+#  Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
```
Comment (shahmishal) [inline swift-ci/sdks/static-linux/scripts/fetch-source.sh]: We should update the year

Comment (al45tair) [reply swift-ci/sdks/static-linux/scripts/fetch-source.sh]: Fixed. I've added copyright banners to various other files too.

**Comment by al45tair** [comment]
@swift-ci test

**Review by shahmishal — APPROVED** [review]

**Pull request merged**